src/app/OperationalDeviceProxy.cpp

```diff
@@ -57,10 +58,35 @@ void OperationalDeviceProxy::MoveToState(State aTargetState)
     }
 }
 
+bool OperationalDeviceProxy::AttachToExistingSecureSession()
+{
+    VerifyOrReturnError(mState == State::NeedsAddress || mState == State::Initialized, false);
+
+    ScopedNodeId peerNodeId(mPeerId.GetNodeId(), mFabricInfo->GetFabricIndex());
+    auto sessionHandle = mInitParams.sessionManager->FindSecureSessionForNode(peerNodeId, Transport::SecureSession::Type::kCASE);
```

Comment:
It's unfortunate we should need this, because if we are tracking PASE sessions by node ID, these should be keyed on the Temporary Local Node ID range.  The entire point of that range is that it can't collide with other IDs, e.g. operational node IDs for CASE.

But I understand that for now, we do have PASE sessions keyed on operational node ID, so this is necessary.